raiden/transfer/node.py

```diff
@@ -765,6 +765,7 @@ def handle_state_change(
         )
     elif type(state_change) == ActionInitChain:
         assert isinstance(state_change, ActionInitChain), MYPY_ANNOTATION
+        at_init = True
```

Comment:
So `chain_state` the paramter at this point is `None`. Wouldn't it be cleaner to set `chain_state = iteration.new_state` after the `handle_chain_init`? we won't need the `at_init` variable in this case

Reply:
hm indeed. I think you are right. I will amend that commit.